### ISSUE TITLE
Question about consistency loss

### ISSUE THREAD

**Issue by Le-ying** [SmiletoE/HPAL#2]
Hello @SmiletoE ,

I've noticed that a consistency loss is part of the training loss in your code, which is not mentioned in the paper. I wonder if the consistency loss is used to achive the results provided in the paper.

Could you please provide some guidance?

Thank you for your time.

**Comment by SmiletoE** [comment]
Hi @Le-ying,
Our paper mainly focuses on the active learning part i.e. HMMU and FDS modules. Thus, for the semi-supervised part, we only built a commonly used teacher-student model for unlabelled data usage, which is not detailed in the paper. The consistency loss and pseudo-label loss are both part of the teacher-student model, and you can easily find lots of relevant information online.
In fact, the teacher-student (TS) model we used is pretty basic, and its performance is not very good. There are some recent semi-supervised methods that outperform TS, and you can even replace TS with them and probably get better results.
